server/plugin/command.go

```diff
@@ -785,3 +794,24 @@ func SliceContainsString(a []string, x string) bool {
 	}
 	return false
 }
+
+func (p *Plugin) getExcludedRepo(args *model.CommandArgs) (string, error) {
+	subs, err := p.GetSubscriptionsByChannel(args.ChannelId)
+	if err != nil {
+		return "", err
+	}
+
+	var repoMatch []string
+	if len(subs) != 0 {
+		for _, sub := range subs {
+			if len(sub.Flags.ExcludedRepos) > 0 {
+				repoMatch = strings.Split(strings.Trim(strings.Join(sub.Flags.ExcludedRepos, ", "), "/"), "/")
+			}
+		}
+		if len(repoMatch) == 0 {
+			return "", nil
+		}
+		return repoMatch[1], nil
+	}
+	return "", nil
```

Comment:
This seems a bit off. Should we add a `break` after the `repoMatch =` line?

Or or it could be written this way:

```suggestion
for _, sub := range subs {
    if len(sub.Flags.ExcludedRepos) > 0 {
        repoMatch = strings.Split(strings.Trim(strings.Join(sub.Flags.ExcludedRepos, ", "), "/"), "/")
        if len(repoMatch) > 1 {
            return repoMatch[1], nil
        }
    }
}

return "", nil
```

Reply:
ok, i updated as suggest